letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -882,15 +881,31 @@ def _enable_redirect(self, ssl_vhost, unused_options):
                         "redirection")
             self._create_redirect_vhost(ssl_vhost)
         else:
-            # Check if redirection already exists
-            self._verify_no_redirects(general_vh)
+            # Check if LetsEncrypt redirection already exists
+            self._verify_no_letsencrypt_redirect(general_vh)
+
+
+            # Note: if code flow gets here it means we didn't find the exact
+            # letsencrypt RewriteRule config for redirection. So if we find
+            # an other RewriteRule it may induce a loop / config mismatch.
```

Comment:
Maybe better to say "this is likely to be fine in most or all cases, but redirect loops are possible in very obscure cases; see https://github.com/letsencrypt/letsencrypt/issues/1620 for reasoning".